data/en/mbs_0205.json

```diff
@@ -0,0 +1,449 @@
+                                    },
+                                    {
+                                        "q_code": "146c",
+                                        "value": "Special/calendar events",
+                                        "label": "Special/calendar events"
+                                    },
+                                    {
+                                        "q_code": "146d",
+                                        "value": "Weather",
+                                        "label": "Weather"
+                                    },
+                                    {
+                                        "q_code": "146e",
+                                        "value": "Price effects",
+                                        "label": "Price effects"
+                                    },
+                                    {
+                                        "q_code": "146f",
+                                        "value": "Currency effects (increase/decrease in the currency value)",
+                                        "label": "Currency effects (increase/decrease in the currency value)"
+                                    },
+                                    {
+                                        "q_code": "146g",
+                                        "value": "Other",
+                                        "label": "Other"
+                                    }
+                                ],
+                                "id": "changes-in-turnover-answer-2",
+                                "label": "",
+                                "description": "",
+                                "type": "Checkbox",
+                                "mandatory": false
+                            }],
+                            "type": "General",
+                            "id": "changes-in-turnover-question-2"
+                        }],
+                        "title": "Changes in turnover"
+                    },
+                    {
+                        "type": "Question",
+                        "id": "changes-in-turnover-block-3",
+                        "title": "Changes in turnover",
+                        "questions": [{
+                            "type": "General",
+                            "id": "changes-in-turnover-question-3",
+                            "title": "Please describe the changes in turnover for {{respondent.trad_as_or_ru_name}} in more detail",
+                            "answers": [{
+                                "guidance": {
+                                    "hide_guidance": "Hide examples of commentary on changes to turnover",
+                                    "show_guidance": "Show examples of commentary on changes to turnover",
+                                    "content": [{
+                                            "list": [],
```

Comment:
can delete these empty lists?

Reply:
I thought that would fail the schema validation but you're right. They aren't needed.